changelog/28042.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+activity: The sys/internal/counters/activity endpoint will return current month data when the start_date parameter is set to a future date.
```

Comment:
```suggestion
activity: Fixed issue where sys/internal/counters/activity endpoint would not return current month data when the end_date parameter is set to a future date.
```

Reply:
Some of this is subjective wording, but it should be `end_date`, not `start_date`. Feel free to take just the `end_date` fix